### DISCUSSION TITLE
news/release-9.1/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#634] [News]
# news/release-9.1/

Oops

https://prismlauncher.org/news/release-9.1/

**Comment by ka-xa-ta** [comment]
meow

**Comment by dah950** [comment]
I can't launch any versions that don't use java 8

**Comment by advnpzn** [comment]
Does one have to set the JRE to use Nvidia graphics in windows settings to make mc instances launched through prism launcher to use nvidia GPU ?
